### PR TITLE
add check for clobber keyword in obsparams file

### PR DESCRIPTION
**Pull request by jburba**
I modified the `hera-sim-vis.py` script to include a check for the clobber keyword in the filing parameters of the obsparam file.  If the clobber keyword is not present in the obsparams file, then clobber is set to False.  Otherwise, clobber is set to the value in the obsparam file.

### CODE REVIEW

File: scripts/hera-sim-vis.py [hunk 1]
```diff
@@ -165,10 +165,14 @@ def cprint(*args, **kwargs):
         outfile = (
             base_path / f"{cfg_filing['outfile_name']}.{cfg_filing['output_format']}"
         )
+        if "clobber" not in cfg_filing:
+            clobber = False
```
Comment (piyanatk) [inline scripts/hera-sim-vis.py:169]: Maybe also print a warning here that `clobber` is being set to `False`.

**Review by piyanatk — APPROVED** [review]
This looks good @jburba. Just one suggestion but I approved.

File: scripts/hera-sim-vis.py (hunk 1 above)
Comment (steven-murray) [reply scripts/hera-sim-vis.py:169]: I don't think we'll need the warning. If clobber is False, and the file already exists, then an error will be raised later anyway, right? I think setting to False is non-destructive and so doesn't need a warning.

Comment (steven-murray) [reply scripts/hera-sim-vis.py:169]: To make the code slightly simpler, could remove all this if/else, and just replace the line 171 with `clobber=cfg_filing.get('clobber', False)`

Comment (jburba) [reply scripts/hera-sim-vis.py:169]: Yes, pyuvdata raises an error if the file exists and clobber is set to False.  And I forgot about the get function.  I'll make that change.

**Pull request merged**